common/containers/App/Notifications.jsx

```diff
@@ -0,0 +1,63 @@
+// @flow
+import React from 'react';
+import { connect } from 'react-redux';
+import { closeNotification } from 'actions/notifications';
+import type { Notification } from 'actions/notifications';
+
+function NotificationRow(props: {
+    notification: Notification,
+    onClose: (n: Notification) => void
+}) {
+    const { msg, level } = props.notification;
+    let klass = '';
+
+    switch (level) {
+        case 'danger':
+            klass = 'alert-danger';
+            break;
+        case 'success':
+            klass = 'alert-success';
+            break;
+        case 'warning':
+            klass = 'alert-warning';
+            break;
+    }
+
+    return (
+        <div className={`alert popup ${klass} animated-show-hide`} role="alert" aria-live="assertive">
+            <span className="sr-only">{level}</span>
+            <div className="container" dangerouslySetInnerHTML={{ __html: msg }} />
+            <i
+                tabIndex="0"
+                aria-label="dismiss"
+                className="icon-close"
+                onClick={() => props.onClose(props.notification)}
```

Comment:
The reason you're using `() => ....` is just to pass `props.notification` along and not have it bind `this` in the `onClose` function, correct?